csharp/codeql-extractor.yml

```diff
@@ -16,3 +16,10 @@ file_types:
     extensions:
       - .cs
 legacy_qltest_extraction: true
+options:
+  compression:
```

Comment:
I prefer `trap_compression`.

Reply:
Will use groups instead as discussed in slack.